setup.sh

```diff
@@ -103,694 +144,259 @@ vercomp () {
-  write "  ./setup.sh --help"
-  writeBold "* You can start Kuzzle by typing:"
-  write "  docker-compose -f $COMPOSER_YML_PATH up -d"
-  writeBold "* You can see the logs of the Kuzzle stack by typing:"
-  write "  docker-compose -f $COMPOSER_YML_PATH logs -f"
-  writeBold "* You can check if everything is working by typing:"
-  write "  curl -XGET http://localhost:7512/"
-  writeBold "* You can stop Kuzzle by typing:"
-  write "  docker-compose -f $COMPOSER_YML_PATH stop"
-  writeBold "* You can restart Kuzzle by typing:"
-  write "  docker-compose -f $COMPOSER_YML_PATH restart"
-  writeBold "* You can read the docs at http://docs.kuzzle.io/"
+  write_success "[✔] Kuzzle is now running."
   echo
 }
 
-writeHelpToFile() {
-  echo "" > $README_PATH
-  writeLnFile "# Kuzzle Short Help"
-  writeLnFile "  ================="
-  writeLnFile ""
-  writeLnFile "* You can start Kuzzle by typing:"
-  writeLnFile "  docker-compose -f $COMPOSER_YML_PATH up -d"
-  writeLnFile "* You can see the logs of the Kuzzle stack by typing:"
-  writeLnFile "  docker-compose -f $COMPOSER_YML_PATH logs -f"
-  writeLnFile "* You can check if everything is working by typing:"
-  writeLnFile "  curl -XGET http://localhost:7512/"
-  writeLnFile "* You can stop Kuzzle by typing:"
-  writeLnFile "  docker-compose -f $COMPOSER_YML_PATH stop"
-  writeLnFile "* You can restart Kuzzle by typing:"
-  writeLnFile "  docker-compose -f $COMPOSER_YML_PATH restart"
-  writeLnFile "* You can read the docs at http://docs.kuzzle.io/"
+the_end() {
+  write_info "You can see the logs of the Kuzzle stack by typing:"
+  write " docker-compose -f $COMPOSE_YML_PATH logs -f"
+  write_info "You can stop Kuzzle by typing:"
+  write " docker-compose -f $COMPOSE_YML_PATH stop"
+  write_info "You can start Kuzzle by typing:"
+  write " docker-compose -f $COMPOSE_YML_PATH up -d"
+  write_info "You can restart Kuzzle by typing:"
+  write " docker-compose -f $COMPOSE_YML_PATH restart"
+  write
+  write_info "You can read the docs and start following the getting started at"
+  write_title "https://docs.kuzzle.io/#sdk-play-time"
 }
 
-# Main execution routine
-# ===========================
+######### MAIN
 
 if [ "$1" == "--help" ]; then
-  echo
-  writeBold "# Kuzzle short help"
-  writeBold "  ================="
-  shortHelp
-  exit 0
+  echo "--help     show this help"
+  echo "--no-run   only install Kuzzle, don't run it"
+  exit 1
 fi
 
+if [ ! -f "${KUZZLE_DIR}/uid" ]; then
+  echo $(LC_CTYPE=C tr -dc A-Fa-f0-9 < /dev/urandom | fold -w 64 | head -n 1) > "${KUZZLE_DIR}/.uid"
+  FIRST_INSTALL=1
+fi
+
+ANALYTICS_UUID=$(cat "${KUZZLE_DIR}/.uid")
 clear
 echo
-writeBold "# Kuzzle Setup"
-writeBold "  ============"
+write_title "# Kuzzle Setup"
+write_title "  ============"
 echo
-writeBold "This script will help you launching Kuzzle and installing"
-writeBold "all the necessary dependencies."
+write "This script will help you running Kuzzle and installing"
```

Comment:
Proposal: `This script will get a Kuzzle up and running, along with all the necessary dependencies`

Reply:
Not true... It will not get the dependencies up and running. It will first check the dependencies are installed and up and running. And, if the requirements are met, it will get a Kuzzle up and running...